index.js

```diff
@@ -1,9 +1,9 @@
 require('dotenv').config()
-const { Octokit } = require('@octokit/rest')
-
-const checkInterval = 1000 // ms
-const milestoneStep = 500
+const { Octokit } = require('@octokit/rest') // Requires octokit
```

Comment:
I dont really see the point in most of those comments. The majority just state the obvious like "water is wet"

Reply:
Fair enough, I usually don’t do things like that when commenting my own code